### DISCUSSION TITLE
Ez Keys No sound sends...Windows Server Mac (Logic) Pluging

### DISCUSSION THREAD

**Discussion by charlesramirezhof** [apohl79/audiogridder#552] [General]
Hello everybody.

my usual system is to use a windows as a server and a mac as a plug. It works perfect (I usually use it with the virus and sh101 powercore through jbridge) The question is that something has happened to me that never happened to me with any VST ... With the **ez keys** everything seems to work correctly but does not transmit audio ... it just doesn't sound like I tried other plugs and everything is fine but with this one in particular it is not possible ... Has anyone done a test with it?

a hug to all and as pd I will put that the console1 also works correctly with windows as server and mac as plug

**Comment by apohl79** [comment]
I never tried ez keys. Does it work with 1.1? Or doesn't it work with any version?

**Reply by charlesramirezhof** [reply]
i try it on SERVER Windows 10 pro and 1.2.0-Beta3 and hackintosh Logic Pro x 10.6 and 1.2.0 Pluging.. .if you want i try to install 1.1 server plug to make a try... Is so strange because all looks works fine midi video.... pluging.. .but no sound in it... I tried it just in case in stan alone to see that the libraries give sound and if they sound on windows stand alone plug ... So it seems that the plug is well installed.
If you want me to do a test with this or another plug, do not hesitate to tell me and it will be tested: =))))) I love audiogrider

**Comment by CaBuffalo** [comment]
"if you want i try to install 1.1 server plug to make a try" --- did you
install the AG plugin of 1.1 as well? Please note that plugin 1.2 will not
work with server 1.1

We have the almost same set up here. Which plugin are you using?

On Sat, May 15, 2021 at 12:16 PM charlesramirezhof ***@***.***>
wrote:

> i try it on SERVER Windows 10 pro and 1.2.0-Beta3 and hackintosh Logic Pro
> x 10.6 and 1.2.0 Pluging.. .if you want i try to install 1.1 server plug to
> make a try... Is so strange because all looks works fine midi video....
> pluging.. .but no sound in it... I tried it just in case in stan alone to
> see that the libraries give sound and if they sound on windows stand alone
> plug ... So it seems that the plug is well installed.
> If you want me to do a test with this or another plug, do not hesitate to
> tell me and it will be tested: =))))) I love audiogrider
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/552#discussioncomment-742600>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HFCUNBHQTYXK7IEZ3TN3CALANCNFSM445GUGCA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by datpham** [reply]
I can confirm that EZKeys doesn't work with AG. I've tried all AG versions from 1.1 --> 1.2 beta3 , EzKeys loads up the UI but doesn't seem to output any sound.  I thought something to do with Toontrack VSTi so I loaded Superior Drummer and EZBass and they both work fine with AG except EZKeys 

My setup is a single local Windows 10 (AG server and plugin in the same PC) 
REAPER 6.27
EZkeys version 1.25

BTW, thanks for a wonderful software, AG has now replaced VEP6 in my studio setup.

**Reply by lesbrunn** [reply]
> BTW, thanks for a wonderful software, AG has now replaced VEP6 in my studio setup.

I'm happy for you! This is what I want too, but VEP6 is rock-solid on my systems; it doesn't crash. AG has been problematic for me. Do you have access to multiple outputs from AG to the DAW? Do multiple VI's play well simultaneously without latency/audio breaks? I haven't been able to do that smoothly in Pro Tools. Could it be that different DAWs react differently to AG? I haven't tried server and plugins on the same computer. How do you make the connection? Doesn't that somehow defeat the purpose of lifting the plugin load off your PC and letting a server computer handle it?

**Reply by datpham** [reply]
atm, I don't think AG can handle multi-outputs (eg. Kontakt) yet , but hopefully DEV will improve it in the future. Most of my projects involve virtual instruments for drums, Basses, strings (or horns) and pianos (keys). The rest will be acoustic, electric guitars and vocals with VST FX chains.

My WIn10 PC is an old Xeon X5670  2.93GHz 6 cores (12 Threads). I used to run VEP6 in another Xeon X5670 to get more juice for VTSi/VST but VEP6 crashed alots so later on I switched to AG though still in development and not as fancy as VEP but things getting smoother. Then followed comments from users using AG on the same PC I have tried it and my PC actually can handle a lots more VTSs than before without a need to use another PC. I think this workflow suits my "everyday" projects in term of conveniences 

From the explanation in some comments that even my PC has 6 cores but the DAW (and all those VSTs) will only utilize a single CPU core which maxed up very quickly and causes pops/clicks/noise etc... , with AG in the chain it uses all cores and spreads the load evenly which allows more plugins to be used. 

Of course with heavy projects involve lots of strings/orc instruments I will reach out to another PC (or  even to my server DELL R710 2x Xeon X5675  3.4GHz  12 Cores) to handle those. :)

**Reply by datpham** [reply]
Hi @apohl79  @CaBuffalo 

Is there any chance to investigate the EZkeys issue?  Thanks

BTW, love your works

**Comment by CaBuffalo** [comment]
Hi, datpham,

I don't have EzKey but I have tried EZdrummer. I just tried and It works
fine. I can load it with AG inst. My set up is the same as yours, Windows
Server and Logic DAW.

Does any other instrument work for you?

On Wed, Jun 9, 2021 at 6:26 PM datpham ***@***.***> wrote:

> Hi @apohl79 <https://github.com/apohl79> @CaBuffalo
> <https://github.com/CaBuffalo>
>
> Is there any chance to investigate the EZkeys issue? Thanks
>
> BTW, love your works
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/552#discussioncomment-848488>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DB6WGD6RT5SNO7WPTTSAIDVANCNFSM445GUGCA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by datpham** [reply]
Thanks for reply, I tried all the Toontrack VSTi's work perfectly except EZKeys. As explained previously, AG can load the EZKey interface but no sounds seem to produce.

Cheers
